icubam/db/csv.py

```diff
@@ -0,0 +1,132 @@
+import csv
+import sys
+
+class CSV:
+	""" """
+
+	def __init__(self, db):
+		self.store = db
+
+	def get_default_admin(self):
+		# create default admin
+		return self.store.add_default_admin()
+
+	def import_icus(self,admin_user_id: int, csv_file_path:str, forceUpdate=False):
+		csv_data = csv.reader(open(csv_file_path))
```

Comment:
any reason not to use pandas.read_csv? That would generally be faster for processing that for loops over rows.

Reply:
+1 for a pandas-based solution for maintainability down the line.

Reply:
plus we already have pandas as a dependency

Reply:
i don't know panda(to be honest i don't know too much of python either) I will give it a look

Reply:
simply use `pd.read_csv(csv_file_path)` here instead

Reply:
and `import pandas as pd`

Reply:
Done, it was a bit more than just a line, since iterating trough dataframe is done differently but in the end it's cleaner.